src/Chapter2.hs

```diff
@@ -336,8 +348,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
-
+subList a b x
+  | a<0 || b <0 = []
```

Comment:
That's right!
In Haskell functions and operators are always separated from the arguments with spaces 👾 

```suggestion
  | a < 0 || b < 0 = []
```